php-amqp.yaml

```
@@ -0,0 +1,50 @@
+package:
+  name: php-amqp
+  version: 1.11.0
+  epoch: 0
+  description: "PHP extension to communicate with any AMQP compliant server"
+  copyright:
+    - license: PHP-3.01
+  dependencies:
+    runtime:
+      - php
+      - rabbitmq-c
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - autoconf
+      - busybox
+      - php
+      - php-dev
+      - rabbitmq-c-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/php-amqp/php-amqp
+      tag: "v${{package.version}}"
+      expected-commit: a757b6657eedbbe3802756720f78f33e058d8d66
+
+  - name: Prepare build
+    runs: phpize
+
+  - name: Configure
+    runs: |
+      set -x
+      ./configure
+
+  - uses: autoconf/make
+
+  - name: Make install
+    runs: |
+      set -x
+      INSTALL_ROOT="${{targets.destdir}}" DESTDIR="${{targets.destdir}}" make install
+
+update:
+  enabled: true
+  github:
+    identifier: php-amqp/php-amqp
+    strip-prefix: v
+    tag-filter: v
```

```suggestion
    tag-filter: v
    use-tag: true
```
think adding this will fix the CI check

---

cool, let me try that!